experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/treeNode.ts

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { fail } from "../../../util";
+import { ObjectNodeSchema, AllowedTypes, FieldNodeSchema, MapSchema } from "../../typed-schema";
+import { TreeNode, ObjectNode, FieldNode, MapNode } from "../editableTreeTypes";
+import { SharedTreeObject, SharedTreeList, SharedTreeMap, SharedTreeNode } from "./types";
+
+/** Stores a reference to a {@link TreeNode} on a {@link SharedTreeNode}. */
+const treeNodeSymbol = Symbol("TreeNode");
+
+// This is intentionally a WeakMap, rather than a private symbol (e.g. like `treeNodeSymbol`).
```

Comment:
Can we convert this to a TSDoc comment for consistency / intellisense support?

Reply:
Sure!